src/pairs/GameState.py

```diff
@@ -25,7 +25,7 @@ def fold_next(self):
 
     def game_to_tuple(self):
         curr_deck_state = list(intMapper(self.deck.values(), self.deck.curr_deck.values()))
-        curr_player_states = [intMapper(self.deck.values(), player.hand_state.values())
+        curr_player_states = [intMapper(player.hand_capacity, player.hand_state.values())
```

Comment:
`player.hand_capacity.values()` is probably what you want.

Reply:
fixed thanks